internal/server/data/migrations.go

```diff
@@ -594,7 +596,124 @@ func setDefaultOrgID() *migrator.Migration {
 				}
 			}
 			return nil
+		},
+	}
+}
+
+func addProviderGroups() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-09-07T17:05",
+		Migrate: func(tx migrator.DB) error {
+			// setup the new tables
+			_, err := tx.Exec(
+				`
+					CREATE TABLE IF NOT EXISTS provider_groups (
+						id bigint NOT NULL,
+						created_at timestamp with time zone,
+						updated_at timestamp with time zone,
+						deleted_at timestamp with time zone,
+						provider_id bigint NOT NULL,
+						name text NOT NULL,
+						organization_id bigint
+					);
+					CREATE TABLE IF NOT EXISTS provider_groups_identities (
```

Comment:
is this named right? There should be no link between provider_groups and identities. it should be provider_groups_provider_users ?

Reply:
Fixed to provider_groups_provider_users to more accurately match the model.